nodestream_akamai/akamai_utils/model.py

```diff
@@ -30,24 +34,24 @@ def origin_count(self):
     def hostname_count(self):
         return len(self.hostnames)
 
-    @property
-    def cloudlet_policy_count(self):
-        return {"edgeRedirector": len(self.cloudlet_policies["edgeRedirector"])}
+    # @property
+    # def cloudlet_policy_count(self):
+    #     return {"edgeRedirector": len(self.cloudlet_policies["edgeRedirector"])}
```

Comment:
Shall we remove this?

Reply:
I just left it in for legacy reasons. The now split between regular cloudlets and redirect configs is something I need to discuss with you guys